scripts/test.sh

```diff
@@ -2,16 +2,33 @@
 
 echo "Starting Flame Tester"
 echo "---------------------"
+root_dir=$(pwd)
 for file in $(find . -type d -name "test"); do
   dir=$(dirname $file)
   cd $dir
   echo "Testing $dir"
-  flutter test
+  flutter test --coverage
   test_result=$?
   if [ $test_result -ne 0 ]; then
     exit $test_result
   fi
+
+  if [ -f ".min_coverage" ]; then
+    min_coverage=$(cat .min_coverage)
+
+    coverage_summary=$(lcov --summary coverage/lcov.info)
+    echo "$coverage_summary"
+
+    current_cov=$(echo "$coverage_summary" | grep lines | grep -Eow '\d+\.\d+')
+
+    dart "$root_dir/scripts/check_coverage.dart" "$current_cov" "$min_coverage"
```

Comment:
```suggestion
    dart "$(dirname $0)/check_coverage.dart" "$current_cov" "$min_coverage"
```

Reply:
@spydon I couldn't make the `dirname $0` work. The command behaves like you explained to me but when I use it to create the path to the dart file I want to execute, dart can't find the file because the current session is on a different folder. With pwd I have an absolute path and it seems to be more secure.

But I am open to suggestion here.